<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="m_centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QScienceSpinBox" name="doubleSpinBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonSet">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEditValue"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="m_statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>QScienceSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
